a10networks/a10-octavia
# Feature/flavor support for server option

## Description
Feature/flavor support for server option

## Jira Ticket
N/A

## Technical Approach
 - flavor schema implementation for server options

## Config Changes
**This is only required if the config has been updated in this PR**
- Required: Provide example config snippet within pre block

<pre>
<b>openstack loadbalancer flavorprofile create --name pf1 --provider a10 --flavor-data '{"server":{"conn_limit":65535, "conn_resume": 5000}}'</b>
</pre>


## Test Cases
- create servers with flavorprofile which shows above

## Manual Testing
- create flavor profile with server options
- create loadbalancer listener, port and member
- on thunder see if server configure correctly with those server options in flavor
